e2e/README.md

```diff
@@ -1,23 +1,80 @@
-End to End Tests
-================
+# End to End Tests
 
-This package contains integration tests. 
+This package contains integration tests.
 
-The `terraform` folder has provisioning code to spin up a Nomad cluster on AWS. The tests work with the `NOMAD_ADDR` environment variable which can be set either to a local dev Nomad agent or a Nomad client on AWS. 
+The `terraform` folder has provisioning code to spin up a Nomad cluster on AWS.
+The tests work with the `NOMAD_ADDR` environment variable which can be set
+either to a local dev Nomad agent or a Nomad client on AWS.
 
-Local Development
-=================
-The workflow when developing end to end tests locally is to run the provisioning step described below once, and then run the tests as described below.
-When making local changes, use `./bin/update $(which nomad) /usr/local/bin/nomad` and `./bin/run sudo systemctl restart nomad` to destructively modify the provisioned cluster.
+## Local Development
 
-Provisioning
-============
-You'll need AWS credentials (`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`) to create the Nomad cluster. See the [README](https://github.com/hashicorp/nomad/blob/master/e2e/terraform/README.md) for details. The number of servers and clients is configurable, as is the configuration file for each client and server.
+The workflow when developing end to end tests locally is to run the
+provisioning step described below once, and then run the tests as described
+below.
 
-Running
-===========
-After completing the provisioning step above, you should see CLI output showing the IP addresses of Nomad client machines. To run the tests, set the NOMAD_ADDR variable to `http://[client IP]:4646/`
+When making local changes, use `./bin/update $(which nomad) /usr/local/bin/nomad`
+and `./bin/run sudo systemctl restart nomad` to destructively modify the
+provisioned cluster.
 
+## Provisioning Test Infrastructure on AWS
+
+You'll need Terraform and AWS credentials (`AWS_ACCESS_KEY_ID` and
+`AWS_SECRET_ACCESS_KEY`) to create the Nomad cluster. See the
+[README](https://github.com/hashicorp/nomad/blob/master/e2e/terraform/README.md)
+for details. The number of servers and clients is configurable, as is the
+configuration file for each client and server.
+
+## Provisioning
```

Comment:
How about `## Provisioning e2e Framework Nomad Cluster` - to help point out this is a separate provisioning step from the TF infra provisioning that happens above